sm/plugins/tedge_apt_plugin/tests/main.rs

```diff
@@ -0,0 +1,167 @@
+        ; "right path with wrong version"                                                                       // description
+    )]
+    #[test_case(
+        &format!("install {} --file {} --module-version {}", ROLLDICE, "notapackagepath", "someversion"),       // input
+        &format!("ERROR: Parsing Debian package failed"),                                                       // expected stderr
+        5                                                                                                       // expcted exit code
+        ; "wrong path with wrong version"                                                                       // description
+    )]
+    fn install_from_local_file_fail(
+        input_command: &str,
+        expected_stderr: &str,
+        expected_exit_code: i32,
+    ) -> Result<()> {
+        // no setup needed, wrong arguments are provided to tedge apt plugin
+        let (stdout, stderr, exit_code) = run_cmd(TEDGE_APT_COMMAND, input_command)?;
+
+        // asserting command failed
+        assert_that!(stdout, is(none()));
+        assert_that!(stderr.unwrap(), match_regex(expected_stderr));
+        assert_that!(exit_code, eq(expected_exit_code));
+        Ok(())
+    }
+
+    #[test_case(
+        &format!("install {} --file {}", ROLLDICE, ROLLDICE_FILE_PATH),                                         // input
+        &format!("The following NEW packages will be installed\n  {}", ROLLDICE),                               // expected stdout
+        0                                                                                                       // expected exit code
+        ; "path"                                                                                                // description
+    )]
+    #[serial]
+    #[test_case(
+        &format!("install {} --file {} --module-version {}", ROLLDICE, ROLLDICE_FILE_PATH, ROLLDICE_VERSION),   // input
+        &format!("The following NEW packages will be installed\n  {}", ROLLDICE),                               // expected stdout 
+        0                                                                                                       // expected exit code
+        ; "path with version"                                                                                   // description
+    )]
+    #[serial]
+    #[test_case(
+        &format!("install {} --module-version {} --file {}", ROLLDICE,  ROLLDICE_VERSION, ROLLDICE_FILE_PATH),  // input
+        &format!("The following NEW packages will be installed\n  {}", ROLLDICE),                               // expected stdout 
+        0                                                                                                       // expected exit code
+        ; "version with path"                                                                                   // description
+    )]
+    #[serial]
+    fn install_from_local_file_sucess(
```

Comment:
Interestingly, I would expect these tests to fail in our CI as they are not called with `sudo`, but for some reason they are green.
On my machine these actually fail unless I add the target config as advised in the description.